.github/workflows/publish-release.yml

```diff
@@ -1,77 +1,115 @@
-name: Publish release builds
-
+# GitHub Actions workflow to publish new releases.
+name: "Release"
```

Comment:
Currently it's:

```
Release / Release
Release / Gradle Publish
Release / Docker Push
```

Maybe it'd be better:

```
Publish releases / GitHub
Publish releases / Maven
Publish releases / Docker
```

Reply:
Agreed